examples/twodnavierstokes_stochasticforcing.jl

```diff
@@ -125,23 +130,28 @@ nothing # hide
 
 function computetendencies_and_makeplot(prob, diags)
   TwoDNavierStokes.updatevars!(prob)
-  E, D, W, R = diags
+  E, D, W, R, Z, DZ, WZ, RZ = diags
 
   clocktime = round(μ*cl.t, digits=2)
 
   i₀ = 1
   dEdt_numerical = (E[(i₀+1):E.i] - E[i₀:E.i-1])/cl.dt #numerical first-order approximation of energy tendency
+  dZdt_numerical = (Z[(i₀+1):Z.i] - Z[i₀:Z.i-1])/cl.dt #numerical first-order approximation of enstrophy tendency
   ii = (i₀):E.i-1
   ii2 = (i₀+1):E.i
 
   t = E.t[ii]
   dEdt_computed = W[ii2] - D[ii] - R[ii]        # Stratonovich interpretation
-  
-  residual = dEdt_computed - dEdt_numerical
+  dZdt_computed = WZ[ii2] - DZ[ii] - RZ[ii]
+
+  residual_E = dEdt_computed - dEdt_numerical
+  residual_Z = dZdt_computed - dZdt_numerical
+
+  εZ = ε*(forcing_wavenumber^2)  # Estimate of mean enstrophy injection rate
```

Comment:
This is correct since the forcing is "narrow-ish" in wavenumber space.
A better way is:
```julia
εZ= parsevalsum(forcing_spectrum/2, g)/(g.Lx * g.Ly)
```
right?

Reply:
Good suggestion! Changed it